### PR TITLE
update embeddings example

### PR DESCRIPTION
**Pull request by ADefWebserver**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -237,7 +237,8 @@ Creates an embedding vector representing the input text.
 
 ```csharp
 var api = new OpenAIClient();
-var result = await api.EmbeddingsEndpoint.CreateEmbeddingAsync("The food was delicious and the waiter...");
+var model = await api.ModelsEndpoint.GetModelDetailsAsync("text-embedding-ada-002");
+var result = await api.EmbeddingsEndpoint.CreateEmbeddingAsync("The food was delicious and the waiter...", model);
```
Comment (StephenHodgson) [inline README.md:241]: @ADefWebserver did you have trouble with the default model it's trying to use?

Comment (ADefWebserver) [reply README.md:241]: What did not work was:

`var result = await api.EmbeddingsEndpoint.CreateEmbeddingAsync("The food was delicious and the waiter...", Model.Ada)`

It took me awhile to figure out how I could follow the directions from here: https://openai.com/blog/new-and-improved-embedding-model and use this library. So it's not an 'error' just a 'clarification'.

Also, I spent a week coming up with an end-to-end embeddings example: https://github.com/SyncfusionSuccinctlyE-Books/OpenAI-Succinctly/blob/main/Pages/EmbeddingExample.razor

Comment (StephenHodgson) [reply README.md:241]: Very cool!

Thanks for the feedback. I think a good proper solution might be to add some additional checks in the method to ensure the model is right for embedding, and also adding more default static models to the `Model` class. 

Thanks for the updated documentation, I'll knock out the other stuff later. Cheers!

**Review by StephenHodgson — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ADefWebserver) [reply README.md:241]: If you added the checks for the model embedding, what will happen if OpenAI changes things? Also, embeddings are cool, but, for me, were hard to understand. My sample is MIT license so use whatever you need for your samples if you feel they can help others :)

Comment (StephenHodgson) [reply README.md:241]: I may add in future. Have not added anything yet. OpenAI will just tell you which one to use.

**Pull request merged**